test/Azure.Functions.Cli.Tests/E2E/Helpers/ProcessHelper.cs

```diff
@@ -76,6 +78,37 @@ public static void TryKillProcessForPort(string port)
             }
         }
 
+        public static int GetAvailablePort(int minPort = 7000, int maxPort = 8000)
+        {
+            for (int port = minPort; port <= maxPort; port++)
```

Comment:
I'm assuming it won't take long to find an open port, but do we expect many ports to be taken up?

Reply:
There are only 20 tests so at most 20 ports will be taken up